.github/workflows/tests.yaml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [3.5, 3.6, 3.7, 3.8, 3.9]
+        python: [3.6, 3.7, 3.8, 3.9]
```

Comment:
Technically, this would be a different PR; essentially, you're removing support for py35 from the library which would be mixed in with this commit.

Reply:
I removed this because github action ( testing ) failed.

Reply:
thanks and i updated top comment

Reply:
> I removed this because github action ( testing ) failed.

Okay, I now see *why* the github action failed.  i.e. because github themselves have removed the py35 options.  We could work around that, but there's probably no point now.  It's fine to remove 3.5 from the github workflow actions.  Please could you explain a bit more *why* you are removing things? e.g. "I removed this because the github action failed because github have removed py35 support from their default workflows."